src/app/components/bibliography/bibliography.component.html

```diff
@@ -0,0 +1,19 @@
+<div class='biblSpace'>
+  <div class='biblContent'>
+    <span>
+      <span *ngIf="isComplete" class="biblSortingOption">
+        <label> Sort By </label>
```

Comment:
I would like to. Where can I get the selected language? I checked `app.config` and `main-menu`. In my opinion this could be the method in which the selected language revolves around
```
// LANGUAGE
selectLanguage(event: MouseEvent, languageSelected: Language) {
    event.stopPropagation();
    this.translate.use(languageSelected.code);
    this.itemClicked.emit('language');
}
```
Quite right?

Reply:
Please read the README file, section `### 4.2 - Localization`: you will find quite an exhaustive explanation!

Reply:
Implemented. Damn, it's phenomenal!